### DISCUSSION TITLE
Ghidra future

### DISCUSSION THREAD

**Discussion by b00tkitism** [NationalSecurityAgency/ghidra#5816] [Q&A]
Hi
Just wanted to ask what do you think about the future of Ghidra ?
Is it worth to spend time to using and learning it ? Or some monsters like IDA and Binja make Ghidra not useful as it could be ? (By their rich features)
Or maybe one day Ghidra become better than the other tools(Ida, Binja)

**Comment by graps01** [comment]
Based on the active users and open source nature of Ghidra, prospects look great for the community support. While IDA Pro is paid support and Binary Ninja has various tier levels of paid options, Ghidra is free.
Functionality and community following for Ghidra will continue to influence its development. Of course, there can be nothing totally free (underneath all the discourse and effort), but donations may be brought up now and then.

**Comment by jackwpa** [comment]
Here at our pentesting and malware analysis firm, our workflow for static code analysis heavily relies on:

- GHIDRA for exotic stuff, old microcontroller code for example, and sometimes x86 
- JEB Pro for Android code obviously, but also Arm binaries, smart contracts (although the latter is super rare), and other exotic things not handled by GHIDRA, such as wasm
- IDA Pro for x86 code, although JEB does the job well most of the time

I don't think one single tool can do all the jobs.
